### DISCUSSION TITLE
Error in planning

### DISCUSSION THREAD

**Discussion by Harry-Sliseren** [frePPLe/frepple#566] [Q&A]
I'll test  DEMO verion  frepple. Time to time, during the planning process get errors 1, 11.... 
Where could find  meaning and troubleshoot them ?

**Comment by Harry-Sliseren** [comment]
For example result: unplanned with error 255, log file in attachment.
[log_error255.txt](https://github.com/frePPLe/frepple/files/9952899/log_error255.txt)

**Comment by jdetaeye** [comment]
This looks like a crash.  
If you change the parameter "plan.loglevel" to 2, you'll get more detail in the log file which should (hopefully) allow you to narrow down the problematic piece of data and/or model mistake.

**Comment by Harry-Sliseren** [comment]
Thanks for the previous answer!

I have a problem with the fact that when performing planning with various restrictions, sometimes log - files disappear. How could this be solved? 
As i right undeqrstand I need to correct "runplane" from distribution, is it so?

**Comment by jdetaeye** [comment]
>> I have a problem with the fact that when performing planning with various restrictions, sometimes log - files disappear. 

It's a feature to protect you from filling up your disk with old and big log files.  
Your /etc/frepple/djangosettings.py file has a setting MAXTOTALLOGFILESIZE to control the disk space used by the log files. When the logs exceed this limit (200MB by default) we automatically delete the old ones.
Just increase the limit and you can keep gigabytes of logs...

**Comment by Harry-Sliseren** [comment]
Thank you for the previous answer!

I'd like to know about errors such as: 1; 9; 11; 255 etc.
This document (FrePPle\pgsql\doc\postgresql\html\errcodes-appendix.html) presents the PosgreSQL errors, as well as thier names. Is it possible to find something similar with the errors that already mentioned above?

**Comment by jdetaeye** [comment]
> Is it possible to find something similar with the errors that already mentioned above?

It's too simplistic to catch every possible error in just a simple number. We try to do our best in providing a good description of the error and its context in the log file.  
For crashes like the one you're running into that's not possible. It needs to be debugged in depth to find the root cause and prevent it by echoing a good message...
